src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Manufacturer/Blocks/View/products.html.twig

```diff
@@ -0,0 +1,128 @@
+{#**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<div class="card">
+  <h3 class="card-header">
+    {{ 'Products'|trans({}, 'Admin.Global') }}
+    ({{ viewableManufacturer.manufacturerProducts|length }})
+  </h3>
+  <div class="card-body">
+    {% for product in viewableManufacturer.manufacturerProducts %}
+      <div class="card">
+        <div class="card-header clearfix">
+          {% if product.combinations is not empty %}
+            <a href="{{ path('admin_product_form', {'id': product.id}) }}">{{ product.name }}</a>
+          {% else %}
+            {{ product.name }}
+          {% endif %}
+
+          <div class="d-inline-block float-right">
+            <a class="btn btn-primary btn-sm" href="{{ path('admin_product_form', {'id': product.id}) }}">
+              {{ 'Edit'|trans({}, 'Admin.Actions') }}
+            </a>
+            <button type="button"
+                    class="btn btn-primary btn-sm js-form-submit-btn"
+                    data-form-submit-url="{{ path('admin_product_unit_action', {'action': 'delete', 'id': product.id}) }}"
+                    data-form-confirm-message="{{ '%s%s?'|format('Delete item #'|trans({}, 'Admin.International.Feature'), product.id) }}"
```

Comment:
This works 🤔 ? `%s%s` ?

Reply:
Yup. I don't like it (I'd prefer `Delete item #%s?` string instead), but this string is translated and was used in legacy so I decided to keep it.